### PR TITLE
add appveyor for windows-ci-build

### PR DESCRIPTION
**Pull request by chrishelgert**
appveyor will run the ci on windows, so we can make sure, that everything is working on unix and windows.

### CODE REVIEW

File: .appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+environment:
+  matrix:
+    - nodejs_version: "6"
+    - nodejs_version: "7"
+
+install:
+  - ps: Install-Product node $env:nodejs_version
+  - yarn install
```
Comment (danielbayerlein) [inline .appveyor.yml]: Why do you install `yarn`?

**Review by danielbayerlein — CHANGES_REQUESTED** [review]

File: .appveyor.yml (hunk 1 above)
Comment (chrishelgert) [reply .appveyor.yml]: mh? i do not install yarn, the install block will also install the dependencies i think.
i looked at the appveyor-files from fb-projects

**Review by danielbayerlein — APPROVED** [review]

**Review by danielbayerlein — APPROVED** [review]

**Comment by danielbayerlein** [comment]
@chrishelgert I approved these changes. You can merge the PR, I have no authority to do this 😉

**Comment by chrishelgert** [comment]
Now you have 👍🏻

**Pull request merged**